src/Vm.sol

```diff
@@ -138,12 +138,50 @@ interface VmSafe {
     // struct json = { uint256 a; address b; }
     // If we defined a json struct with the opposite order, meaning placing the address b first, it would try to
     // decode the tuple in that order, and thus fail.
```

Comment:
The comments from here and above are for `readJson` right? Let's organize this a bit, since right now the layout is:
- read JSON cheatcodes
- comments on reading JSON
- comments on writing JSON
- write JSON cheatcodes

We should change it to:
- comments on reading JSON
- read JSON cheatcodes
- comments on writing JSON
- write JSON cheatcodes